brayns/io/algorithms/MetaballsGenerator.h

```diff
@@ -44,6 +44,7 @@ class MetaballsGenerator
      * @param threshold Points in 3D space that fall below the threshold
      *        (when run through the function) are ONE, while points above the
      *        threshold are ZERO
+     * @param materials
```

Comment:
Missing doc?

Reply:
yeah, tell me :) you missed the material completely